src/java.xml/share/legal/dom.md

```diff
@@ -1,5 +1,20 @@
 ## DOM Level 3 Core Specification v1.0
 
+### W3C Software Notice
+<pre>
```

Comment:
Since this is a markdown, should this be three back-ticks "```" instead of "<pre>"?

Reply:
The pre tag is supported for code blocks by most markdown implementations so this should be OK and as Joe mentions has been there for ages in this file.